backend/uclapi/workspaces/management/commands/feed_occupeye_cache.py

```diff
@@ -9,8 +13,27 @@ class Command(BaseCommand):
     )
 
     def handle(self, *args, **options):
-        print("Running OccupEye Caching Operation")
-        print("[+] Feeding Cache")
-        cache = OccupeyeCache()
-        cache.feed_cache(full=True)
-        print("Done!")
+        try:
+            print("Running OccupEye Caching Operation")
+            print("[+] Feeding Cache")
+            cache = OccupeyeCache()
+            cache.feed_cache(full=True)
+            print("Done!")
+            incident_name = get_incident_name("Occupeye")
+            if incident_name:
+                delete_incident(incident_name)
+            else:
+                print("No incident present in Cachet!")
+        except Exception as occupeye_error:

```

Comment:
@zipy124 correct me if I'm wrong but I don't think this has been addressed at all in any of the functions mentioned?

Reply:
So it would appear I am an idiot and completely mis-read this review comment and my code.... Sorry after doing the Systems exam paper my brain was frazzeld! Will adress.

Reply:
@zipy124 this is the bit that needs addressing.

Reply:
```
print("Running OccupEye Caching Operation")
print("[+] Feeding Cache")
try:
    cache = OccupeyeCache()
    cache.feed_cache(full=True)
except:
    incident_name = get_incident_name("Occupeye")
    if not incident_name:
        print("Could not find appropriate incident in Cachet!")
        return

    try:
        create_incident(occupeye_error, incident_name)
    except CachetException as cachet_error:
        print(f"Failed to create cachet incident. "
                f"Reason: {repr(cachet_error)}")
    except Exception as cachet_error:
        print(f"Unexpected: Failed to create cachet incident. "
                f"Reason: {repr(cachet_error)}")
else:
    incident_name = get_incident_name("Occupeye")
    if not incident_name:
        print("Could not find appropriate incident in Cachet!" )

    try:
        delete_incident(incident_name)
    except CachetException as cachet_error:
        print(f"Failed to delete cachet incident. "
                f"Reason: {repr(cachet_error)}")
    except Exception as cachet_error:
        print(f"Unexpected: Failed to delete cachet incident. "
                f"Reason: {repr(cachet_error)}")

print("Done!")
```